tools/python/SiblingKiller.py

```diff
@@ -4,7 +4,8 @@
 
```

Comment:
Does it make sense to add `#!/usr/bin/env python3` at the top?

Reply:
In [`INSTALL.md`](https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md) guide there's example how to run tests:

```
../tools/python/run_desktop_tests.py -f . -u ../data/ -d ../data/
```

If remove the first line from `run_desktop_tests.py` then this command will not work.

Reply:
It's not about removing the hashbang from run_desktop_tests.py, it's about adding it into SiblingKiller.py

Reply:
Well, `SiblingKiller.py` should not be run from command line. Only as Python module. So we don’t need `#!/usr/..` as the first line.